wpe/src/main/java/com/wpe/wpeview/WPEView.java

```diff
@@ -42,11 +42,15 @@ public class WPEView extends FrameLayout implements WPESurfaceViewObserver
     private WPESurfaceView m_wpeSurfaceView;
     private FrameLayout m_customView;
 
+    private WPESettings m_settings;
```

Comment:
Here a:
```
private final WPESettings m_settings = new WPESettings();
```

would be better, so you are sure to never forget initializing it in any constructor.

Moreover, it is good to have `final` on this kind of member "containers", it's an interesting pattern for multi-threading programming. In old JVM it also served to optimize bytecode compilation and lifetime of Java objects (I don't know if it is still the case, maybe not thanks to the JIT compilers)